ramls/item.json

```diff
@@ -31,6 +31,10 @@
       "type": "boolean",
       "description": "Records the fact that the record should not be displayed in a discovery system"
     },
+    "displaySummary": {
```

Comment:
Do we need to update samples?

Reply:
I think we can't introduce change to samples